### PR TITLE
ButterflyNetwork Initial

### PR DESCRIPTION
**Pull request by AdiAyyakad**
#### Summary

Initial changes to match current Butterfly styles

#### Reasoning

🦋

### CODE REVIEW

**Comment by AdiAyyakad** [comment]
As per https://github.com/ButterflyNetwork/software/pull/8494#discussion_r277751584:

Let's add a rule to formalize `deprecated` as the prefix for non-reactive values that exist for legacy components.

File: resources/swiftlint.yml [hunk 1]
```diff
@@ -1,40 +1,40 @@
-whitelist_rules:
```
Comment (vineetshah) [inline resources/swiftlint.yml:1]: What is this file used for?

File: resources/xcode_settings.bash [hunk 2]
```diff
@@ -5,7 +5,7 @@ set -e
 defaults write com.apple.dt.Xcode DVTTextEditorTrimTrailingWhitespace -bool YES
 defaults write com.apple.dt.Xcode DVTTextEditorTrimWhitespaceOnlyLines -bool YES
 
-defaults write com.apple.dt.Xcode DVTTextIndentTabWidth -int 2
-defaults write com.apple.dt.Xcode DVTTextIndentWidth -int 2
+defaults write com.apple.dt.Xcode DVTTextIndentTabWidth -int 4
+defaults write com.apple.dt.Xcode DVTTextIndentWidth -int 4
 
-defaults write com.apple.dt.Xcode DVTTextPageGuideLocation -int 100
+defaults write com.apple.dt.Xcode DVTTextPageGuideLocation -int 120
```
Comment (vineetshah) [inline resources/xcode_settings.bash:11]: This is cool but how does this actually get pulled in for use? Should we link to our dev environment setup script?

If that's too much work for now, we should probably just omit this script.

File: README.md [hunk 3]
```diff
@@ -934,6 +989,10 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
+<!-- TODO: DDAssertionFailure -->
+<!-- Not sure I agree with all of this -->
```
Comment (vineetshah) [inline README.md:993]: Did you mean to merge all of your notes into develop?

**Review by AdiAyyakad — COMMENTED** [review]
Closing due to inactivity

**Pull request closed without merging**